s3_scrubber/Cargo.toml

```diff
@@ -22,18 +24,21 @@ serde_json.workspace = true
 serde_with.workspace = true
 workspace_hack.workspace = true
 utils.workspace = true
-
-tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
-chrono = { workspace = true, default-features = false, features = ["clock", "serde"] }
```

Comment:
I like the old style of fitting onto one line. Is there some way to preserve it? Also, having formatting changes as well as semantic ones in the same PR makes reviews harder.

Reply:
I suggest we ignore whitespace in these files until we standardize on a linter -- the style here is just what the most popular VSCode toml plugin does.

Reply:
I've just reverted this and added a newline to the file (Joonas asked for that in an earlier review).